redfish-core/lib/log_services.hpp

```diff
@@ -2741,6 +2835,12 @@ inline void requestRoutesDBusEventLogEntry(App& app)
                     "/redfish/v1/Systems/system/LogServices/EventLog/Entries/" +
                     std::to_string(*id) + "/attachment";
             }
+#ifdef BMCWEB_ENABLE_IBM_MANAGEMENT_CONSOLE
+            asyncResp->res.jsonValue["Oem"]["OpenBMC"]["@odata.type"] =
+                "#OemLogEntry.v1_0_0.LogEntry";
+            asyncResp->res.jsonValue["Oem"]["OpenBMC"]["ManagementSystemAck"] =
```

Comment:
This seems indicating that (https://github.com/ibm-openbmc/bmcweb/pull/472) is needed.

Reply:
i pulled in 472.